app/js/components/orderEntry/ActiveOrders.jsx

```diff
@@ -82,8 +82,18 @@ export class ActiveOrders extends React.Component {
           </td>
           {details}
           <td className="text-center action-padding">
-            <a href="#"> <i className="icon-edit" title="Edit" /> </a>
-            <a href="#"> <i className="icon-remove" title="Delete" /> </a>
+            {
+              (this.props.editOrderNumber === order.orderNumber) ?
+                <p> will REVISE </p> :
```

Comment:
Did you intentionally write the above in capital letters?

Reply:
@dkayiwa yes it was intentional because that is how it was implemented in the previous module.

<img width="650" alt="screen shot 2018-05-03 at 4 09 19 pm" src="https://user-images.githubusercontent.com/19909781/39578287-6c7b528c-4eec-11e8-92ea-bcf217f0ab00.png">